RELEASE_NOTES.md

```diff
@@ -1,3 +1,136 @@
+# ChefDK 4.8
+
+## New Platforms
+
+ChefDK packages are now created for Ubuntu 20.04 and Debian 10! Additionally we've increased package validation for our Windows 10 packages to ensure compatibility. See the [ChefDK Downloads Page](https://downloads.chef.io/chefdk) for a complete list of platforms we produce packages for.
+
+## macOS Binary Signing
+
+Each binary in the macOS ChefDK installation is now signed to improve the integrity of the installation and ensure compatibility with macOS Catalina security requirements.
+
+## Updated Components
+
+### Chef Infra Client 15.11
+
+Chef Infra Client has been updated from 15.7 to 15.11 which includes improvements to resources, additional cookbook helpers, and critical bug fixes for bootstrapping nodes using `knife bootstrap` and SSHing to nodes with `ed25519` keys from Windows hosts. For a complete list of changes, see the [Chef Infra Client 15.11 release notes](https://docs.chef.io/release_notes/#whats-new-in-1511)
+
+### Chef InSpec 4.19
+
+Chef InSpec has been updated from 4.18.51 to 4.19.0. This update includes a large number of fixes to resources and these significant new features:
+
+- You can now develop your own Chef InSpec Reporter plugin and determine how Chef InSpec will report result data. Learn more about Chef Inspec plugins and implementation 2 in our documentation.
+- The inspec archive command packs your profile into a tar.gz file that includes the profile in JSON form as the inspec.json file. Use this JSON file to programmatically examine the profile without needing to load it into Chef InSpec.
+- InSpec accepts a variety of date formats in the waivers.yaml configuration file, rather than only the YYYY-MM-DD format.
+- Use the new inspec command options to control the size of reports:
+    - --reporter-message-truncation sets a length limit for the message field in test failure report data.
+    - --reporter-backtrace-inclusion determines if Ruby backtraces should be included in test failure report data.
+- Implemented VMware and Hyper-V detection on Linux systems
+- Implemented VMware, Hyper-V, Virtualbox, KVM and Xen detection on Windows systems
+- Added helpers virtual_system? and physical_system?.
+
+### Cookstyle 5.23
+
+Cookstyle was upgraded from 5.20 to 5.23 which includes 8 new cops and significant improvements to the detection and autocorrect capabilities in existing cops.
+
+#### New Cops
+
+- ChefModernize/NodeInitPackage
+- ChefDeprecations/WindowsFeatureServermanagercmd
+ - ChefModernize/WindowsRegistryUAC
```

Comment:
@tas50 Are lines 39-43 supposed to be a sub-list under "ChefDeprecations/WindowsFeatureServermanagercmd" ?

Reply:
Fixed